src/views/ModelPlan/Documents/AddDocument/LinkDocument.tsx

```diff
@@ -0,0 +1,416 @@
+
+                    <Field
+                      as={TextInput}
+                      id="FileUpload-LinkDocument"
+                      name="url"
+                    />
+                  </FieldGroup>
+                  <FieldGroup scrollElement="name" error={!!flatErrors.name}>
+                    <Label htmlFor="FileUpload-LinkFileName">
+                      {t('linkDocument.fileNameLabel')}
+                      <RequiredAsterisk />
+                    </Label>
+
+                    <Label
+                      htmlFor="FileUpload-LinkFileName"
+                      hint
+                      className="text-normal text-base margin-y-1"
+                    >
+                      {t('linkDocument.fileNameHelpText')}
+                    </Label>
+
+                    <FieldErrorMsg>{flatErrors.name}</FieldErrorMsg>
+
+                    <Field
+                      as={TextInput}
+                      id="FileUpload-LinkFileName"
+                      name="name"
+                    />
+                  </FieldGroup>
+
+                  <FieldGroup
+                    id="file-type"
+                    scrollElement="documentType"
+                    error={!!flatErrors.documentType}
+                  >
+                    <fieldset className="usa-fieldset margin-top-4">
+                      <legend className="usa-label">
+                        {t('whatType')}
+                        <RequiredAsterisk />
+                      </legend>
+                      <FieldErrorMsg>{flatErrors.documentType}</FieldErrorMsg>
+                      {(Object.keys(DocumentType) as Array<
+                        keyof typeof DocumentType
+                      >)
+                        .filter(documentType => documentType !== 'OTHER') // Filter 'OTHER' to be last in the radio button list
+                        .map(documentType => {
+                          return (
+                            <Field
+                              key={`FileUpload-${documentType}`}
+                              as={RadioField}
+                              checked={values.documentType === documentType}
+                              id={`FileUpload-${documentType}`}
+                              name="documentType"
+                              label={translateDocumentType(
+                                documentType as DocumentType
+                              )}
+                              onChange={() => {
+                                setFieldValue('documentType', documentType);
+                                setFieldValue('otherTypeDescription', '');
+                              }}
+                              value={documentType}
+                            />
+                          );
+                        })}
+                      <Field
+                        as={RadioField}
+                        checked={values.documentType === 'OTHER'}
+                        id="FileUpload-OTHER"
```

Comment:
In case the value of this enum ever changes, we should reference `Other` from the enum proprty, rather than the string property.  Ex: `DocumentType.OTHER` rather than `Other`